app/platform_stats/rest.py

```diff
@@ -106,3 +108,10 @@ def get_usage_for_all_services():
         x['organisation_name'],
         x['service_name']
     )))
+
+
+def postage_description(postage):
+    if postage in UK_POSTAGE_TYPES:
+        return f'{postage} class'
+    else:
+        return 'international'
```

Comment:
Optional: Do we want to break it down further into europe and rest of world here? Or is that even possible? I haven't looked to see if the letter_breakdown returns it broken down into europe and rest of world or just international?

Reply:
It would be broken down into `europe` and `rest-of-world` by default. We want to combine them though so that international letters show up as a single row in [this report](https://www.notifications.service.gov.uk/platform-admin/reports/usage-for-all-services)